pkg/hub/client/policy_merger_test.go

```diff
@@ -18,7 +18,7 @@ func TestPolicyEnforcedClient_mergePolicies(t *testing.T) {
 	tests := []struct {
```

Comment:
It would be great to refactor the tests and prepare some helpers for test policy creation to not have such long test. I don't mean that you should do it in this PR, but it is just a general comment for future